plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer_test.go

```diff
@@ -8,8 +8,11 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/pdata/pcommon"
-	conventions "go.opentelemetry.io/collector/semconv/v1.6.1"
+	deprecatedsemconv "go.opentelemetry.io/collector/semconv/v1.18.0"
+	conventions "go.opentelemetry.io/collector/semconv/v1.22.0"
```

Comment:
similar to previous PR, rather just name this "semconv"

Reply:
Updated